cmd/internal/cli/key_export.go

```diff
@@ -29,7 +29,17 @@ var keyExportSecretFlag = cmdline.Flag{
 	DefaultValue: false,
 	Name:         "secret",
 	ShortHand:    "s",
-	Usage:        "export a secret key",
+	Usage:        "export a secret key (synonym for --private)",
+}
+
+// -p|--private
```

Comment:
I just had a look back at the apptainer implementation, where they added some similar functionality back about a year ago.

https://github.com/apptainer/apptainer/pull/555
https://github.com/apptainer/apptainer/pull/572

There, they added `--public / -p` `--both / -b` flags. The `--public` flag isn't needed here as it's redundant when preserving the default of removing a public key. I don't think we need `--both` at this point, either.

However.. given we will likely merge this into a 3.11 version.. let's drop the `-p` short form for `--private`, so there isn't confusion.

We can properly address this stuff for 4.0 where we can make warranted breaking changes to the CLI, and be clear about any CLI compatibility issues vs apptainer.